pom.xml

```diff
@@ -79,21 +81,39 @@
   </distributionManagement>
 
   <properties>
-    <checkstyle.version>6.1.1</checkstyle.version>
+    <maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
That's just for having it easier to contribute some parts of the code to upstream. Is that an acceptable reason?